templates/taxi/car_detail.html

```diff
@@ -5,8 +5,10 @@
   <h1>Drivers</h1>
   <hr>
   <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-  {% endfor %}
+    {% for driver in car.drivers.all %}
+      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+    {% endfor %}
   </ul>
+  <button><a href="{%  url "taxi:car-confirm-delete" car.id %}">delete</a></button>
```

Comment:
You don't have this endpoint in urls.py
```suggestion
  <button><a href="{%  url "taxi:car-delete" car.id %}">delete</a></button>
```